### PR TITLE
Pretend existing VCRedist is successful install.

### PR DESCRIPTION
**Pull request by gkhanna79**
Port of https://github.com/dotnet/cli/pull/6265.

@Petermarcu PTAL

### CODE REVIEW

**Comment by Petermarcu** [comment]
LGTM

**Comment by Petermarcu** [comment]
Go ahead and merge since it only impacts Windows and we know Mac CI is having trouble today.

File: packaging/windows/sharedframework/bundle.wxs [hunk 1]
```diff
@@ -42,6 +42,7 @@
                        ProductName="$(var.Crt_ProductName)" 
                        Size="$(var.Crt_Size)" 
                        Version="$(var.Crt_Version)" />
+        <ExitCode Behavior="success" Value="1638" />
```
Comment (rakeshsinghranchi) [inline packaging/windows/sharedframework/bundle.wxs]: Burn should not try to install higher version at all. Detect Condition ( https://github.com/gkhanna79/core-setup/blob/2f91187544afc8d39a14f1e181fde394accf1bf2/packaging/windows/sharedframework/bundle.wxs#L32) is expected to return success and skip installation of the item ( in this case VC Redist ). Detection Condition need to be looked deeper in this case.

Comment (gkhanna79) [inline packaging/windows/sharedframework/bundle.wxs]: Evidence indicates that corrupted installation of a version that not in that link is what results in the failure.

**Pull request merged**